### PR TITLE
swift added

### PR DESCRIPTION
**Pull request by JulianKahnert**
Hi,

I added Swift as a `FileType` and also added the build/prod resources files. I used [this Dockerfile](https://github.com/SwiftBlogs/api/blob/master/Dockerfile) as an example.

What do you think @fredlahde and @larsgrah ?

### CODE REVIEW

File: src/main/java/de/worldiety/autocd/util/FileType.java [hunk 1]
```diff
@@ -8,6 +8,7 @@
     VUE("vue", "vue-builder", "vue-prod", "RUN npm run build\n"),
     NUXT("vue", "nuxt-builder", "nuxt-prod", "RUN npm i && npm run build"),
     EISEN("eisen", "vue-builder", "vue-prod", "RUN npm run build\n"),
+    SWIFT("swift", "swift-builder", "swift-prod", ""),
```
Comment (fredlahde) [inline src/main/java/de/worldiety/autocd/util/FileType.java]: Please supply a `defaultBuild` command here as the last argument.

AutoCD has a feature which allows to override the build process specified in the Dockerfile via a `build.sh` file in the project root. As seen [here](https://github.com/worldiety/AutoCD/blob/master/src/main/java/de/worldiety/autocd/docker/DockerfileHandler.java#L151-L157), we use the `defaultBuild` field if we don't find a 'build.sh'.  I reckon that you could move the last line of your `swift-builder` Dockerfile into this field.

**Review by fredlahde — CHANGES_REQUESTED** [review]

File: src/main/java/de/worldiety/autocd/util/FileType.java (hunk 1 above)
Comment (JulianKahnert) [reply src/main/java/de/worldiety/autocd/util/FileType.java]: Hi, I fixed it! Thanks :)

**Review by fredlahde — APPROVED** [review]
Looks good to me :)

**Comment by larsgrah** [comment]
@JulianKahnert Please resolve the remaining conflicts, then this looks good to be merged

**Pull request merged**